.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug Report
+about: Reporting issues with the VA API Platform
+
+---
+
+## APIs Affected
```

Comment:
I am getting nervous about how many places we'll have to updated when we launch new APIs and these getting out of sync. Any way this could still be useful without enumerating the list?  Like could we put in comment like "write the API name here, for example "Facility API"" or something?

Reply:
Sure will do